lib/actions/local/configure.coffee

```diff
@@ -95,100 +67,65 @@ inquirerOptions = (data) -> [
 	}
 ]
 
+joinPath = (path) -> '/' + path.join('/')
```

Comment:
Keep in mind that Windows paths don't start with a slash.

Reply:
Ah, is this Reconfix specific? If so, can we make that clearer, or make the leading slash optional?

Reply:
These paths are used by `resin-image-fs`, which means they are always UNIX style.

Reply:
In that case, it'd be good to add a comment clarifying this here